packages/fields/src/fields/title/index.ts

```diff
@@ -9,13 +9,16 @@ import { __ } from '@wordpress/i18n';
  */
 import type { BasePost } from '../../types';
 import { getItemTitle } from '../../actions/utils';
+import TitleView from './title-view';
 
 const titleField: Field< BasePost > = {
 	type: 'text',
 	id: 'title',
 	label: __( 'Title' ),
 	placeholder: __( 'No title' ),
 	getValue: ( { item } ) => getItemTitle( item ),
```

Comment:
It's nice that this basically means we're reusing the same field for DataForm and DataViews.

Reply:
And actions! For the record, I tested the "Duplicate" action in DataViews, DataForm, and Document Inspector.